templates/rag-dev-bootcamp-mar2024/README.md

```diff
@@ -26,4 +26,17 @@ wget https://anyscale-materials.s3.us-west-2.amazonaws.com/rag-mar-2024.zip
 unzip rag-mar-2024.zip
 ```
 
-2. Navigate to the `01_overview` folder and open the `notebook.ipynb` file to get started with the training program.
\ No newline at end of file
+2. Navigate to the `01_overview` folder and open the `notebook.ipynb` file to get started with the training program.
+
+## Agenda
```

Comment:
10:00 am -> 10:20 am - Set up 
10:20 am -> 10:40 am - RAG overview
10:40 am -> 11:10 am - Canopy
11:10 am -> 11:20 am - Coffee break
11:20 am  -> 12:00 pm - Embeddings pipeline Part 1
12:00 pm -> 1:00 pm - Lunch Break
1:00 pm -> 2:00 pm - Embeddings pipeline Part 2
2:00 pm  -> 3:00 pm - Evaluations Part 1
3:00 pm -> 3:15 pm - Coffee break
3:15 pm -> 4:00 pm - Evaluations Part 2
4:00 pm -> 4:30 pm - Bonus session (stay tuned)


What do you think? Added short break after session 1

Reply:
also change name of the last session.

Reply:
> What do you think? Added short break after session 1

I am ok with it given it signals a mindset switch from canopy to ray.

Just its length is 10 min, whereas the second coffee break is 15 min. Should we stick to 10 min for both ?

Reply:
> 4:00 pm -> 4:30 pm - Bonus session (stay tuned)
I like that it gives us leeway. Is the idea here for instance that we de-risk the service deployments by me trying to run the notebook in the last coffee break and make sure it works. If it doesn't then perhaps we cancel showcasing the deployment ?